### PR TITLE
v0.1.2

### PR DESCRIPTION
**Pull request by mikowitz**
Add tests for feature flags

### CODE REVIEW

File: tests/0.1.2/enabled.yaml [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+  - name: returns the correct value for a simple disabled flag
+    client: feature_flag_client
+    function: enabled
+    input:
+      flag: "feature-flag.simple-disabled"
+    expected:
+      value: false
+
+  - name: always returns false for a non-boolean flag
```
Comment (semanticart) [inline tests/0.1.2/enabled.yaml:26]: My understanding: The scenario here is that given a feature flag that returns a non-boolean (here an integer), calling `enabled` will return `false` regardless of the value.

This gives me pause. I don't think it is wrong but I want to make sure we've thought through it thoroughly.

1. I don't think we want to broach the falsey rules of each supported language.
2. `false` is a better accidental evaluation than `true`, to be sure

I wonder what the user would _expect_ to happen in this case. An exception feels perhaps too weighty and I'm reluctant to add another `raise_if` config -- though that might be the right choice.

🤔 Maybe a warning is the right move?

@jdwyah curious for your thoughts here.

Comment (mikowitz) [reply tests/0.1.2/enabled.yaml:26]: Your understanding is correct, and in the Ruby version (and the Elixir version, based on that), an info-level log is spit out in this case explaining why it's returning false. It also looks like the Java client has the same result, though there's still a `TODO` for the log message. Which is to say, I think your idea of a warning is indeed the correct/already implemented move here.

- Ruby: https://github.com/prefab-cloud/prefab-cloud-ruby/blob/main/lib/prefab/feature_flag_client.rb#L66-L67
- Elixir: https://github.com/prefab-cloud/prefab-cloud-elixir/blob/main/lib/prefab/variant_helpers.ex#L218-L223
- Java: https://github.com/prefab-cloud/prefab-cloud-java/blob/main/src/main/java/cloud/prefab/client/FeatureFlagClient.java#L125-L130

Comment (semanticart) [reply tests/0.1.2/enabled.yaml:26]: Excellent. All good by you, @jdwyah ?

Comment (jdwyah) [reply tests/0.1.2/enabled.yaml:26]: yeah, I think the method asks the question "is it enabled?" and the answer for anything but true is "no".  A warning of "you did something weird" seems 👍

Comment (semanticart) [reply tests/0.1.2/enabled.yaml:26]: Perfect. Thank you both

**Pull request merged**